Shared/Flows/Face/PermissionsViewController.swift

```diff
@@ -142,7 +142,13 @@ class PermissionsViewController: DisclosureModalViewController {
         }
 
         if await UserNotificationManager.shared.getNotificationSettings().authorizationStatus != .authorized {
-            self.notificationSwitchView.state = .disabled
+            if INFocusStatusCenter.default.authorizationStatus == .authorized {
+                self.notificationSwitchView.state = .enabled
+                self.state = .notificationAsk
```

Comment:
fixes issue where we had focus permissions, but weren't enabling the notification ask.